### PR TITLE
Support for symfony 5 (and 4.4 lts)

### PR DESCRIPTION
**Pull request by alekitto**
Minimum symfony version supported is 4.4.

Due to symfony packages breaking changes in 5.0, some breaking changes here are unavoidable.
The vast majority are related to the renamed classes for kernel events, used in typehinting of event listener methods.

The only notable exception is the deletion of the `fazland_api.exception_listener.controller` parameter. The exception controller is now handled by the framework bundle itself under the `framework.error_controller` configuration key. As a result the bundle's `ExceptionListener` was removed in favor of the symfony implementation and the `error_controller` framework key is now correctly set by this bundle if the `catch_exceptions` configuration key is set to `true`.

My proposal is to tag/branch a `1.0` version before this PR is merged to continue support the symfony 4.3-based projects until its EOL and start supporting sf5 projects on dev-master merging this as a starting point for the 2.x version.

### CODE REVIEW

File: tests/Form/Extension/AbstractRequestHandlerTest.php [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+     * @var RequestHandlerInterface
+     */
+    protected $requestHandler;
+
+    /**
+     * @var FormFactory
+     */
+    protected $factory;
+
+    protected $request;
```
Comment (massimilianobraglia) [inline tests/Form/Extension/AbstractRequestHandlerTest.php:35]: missing phpdoc

File: tests/Form/Extension/AbstractRequestHandlerTest.php [hunk 2]
```diff
@@ -0,0 +1,403 @@
...
+
+    protected $serverParams;
```
Comment (massimilianobraglia) [inline tests/Form/Extension/AbstractRequestHandlerTest.php:40]: missing phpdoc

**Review by massimilianobraglia — CHANGES_REQUESTED** [review]
hey there @alekitto! Thank you for your PR

I begun the same path you followed to update the project to Symfony 4.4, so I agree with you to tag the 1.0 version.
For this PR, just left some really minor comments

File: tests/Form/Extension/AbstractRequestHandlerTest.php (hunk 1 above)
Comment (alekitto) [reply tests/Form/Extension/AbstractRequestHandlerTest.php:35]: Fixed

File: tests/Form/Extension/AbstractRequestHandlerTest.php (hunk 2 above)
Comment (alekitto) [reply tests/Form/Extension/AbstractRequestHandlerTest.php:40]: Fixed

**Comment by massimilianobraglia** [comment]
Thank you, @alekitto.

**Pull request merged**